src/cliframe/actionTypes.ts

```diff
@@ -0,0 +1,50 @@
+export type TriCliActionLiteral =
```

Comment:
bunch of boiler plate for cliframe/actions.ts - it really seems like the compiler ought to be able to figure all of this out. :/